backend/lib/components/Auth0.js

```diff
@@ -2,6 +2,7 @@ const axios = require("axios");
 const httpErrors = require("http-errors");
 const qs = require("querystring");
 const { config } = require("../../config");
+const _ = require("lodash");
```

Comment:
What is lodash being used for? I don't see it being used anywhere in the code you just added. If it is not being removed, then we should remove this import.

Reply:
Thanks Manny, it was for updating user's meta data, which was removed then, I will remove this also